todolist/app/screens/App/index.js

```diff
@@ -1,21 +1,26 @@
-import React from 'react';
-import { View, SafeAreaView } from 'react-native';
+import { createStackNavigator, createAppContainer, TabNavigator, createBottomTabNavigator } from 'react-navigation';
 
-import styles from './styles';
-import Title from '../../components/Title';
-import Footer from '../../components/Footer';
-import Input from '../../components/Input';
-import List from '../../components/List';
+import Todo from '../Todo'
+import ListBook from '../ListBook';
 
-function App() {
-  return (
-    <SafeAreaView style={styles.container}>
-      <Title />
-      <Input />
-      <List />
-      <Footer />
-    </SafeAreaView>
-  );
-};
+const Tabs = createBottomTabNavigator({
+  Todo: {
+    screen: Todo
+  },
+  ListBook: {
+    screen: ListBook
+  }
+});
 
-export default App;
+const StackNavigation = createStackNavigator({
+  Home: {
+    screen: Tabs,
+    navigationOptions: {
+      title: 'Training App'
```

Comment:
we usually have another file called navigationOptions.js with all of this (because here you can configure not only the title but all the header also), but in this case, we will leave it like this because there will be just a few